File: flames.c
//flames
#include<stdio.h>
#include<string.h>
int main()
{
    static int i,j,k,l1,l2,l3,count,temp,ex;
    static char a[10],b[10],c[20],d[6]={"flames"};
    //gets(a);
    //gets(b);
    scanf("%s%s",a,b);
    l1=strlen(a);//printf("%d\n",l1);
    l2=strlen(b);//printf("%d\n",l2);
    //hint x=strcmp(b,a);printf("%d\n",x);
    for(i=0;i<l1;i++){
        for(j=0;j<l2;j++){
            if(a[i]==b[j]) {
                a[i]='0';b[j]='0';break;
            }
        } if(a[i]=='0') break;
    }
    for(i=0;i<l1;i++){
        if(a[i]!='0'){
            c[k]=a[i];k++;
        }else{}
    }
    for(j=0;j<l2;j++){
        if(b[j]!='0'){
            c[k]=b[j]; k++;
        }
    }
    //printf("%s\n",c);
    l3=strlen(c);
    //printf("%d\n",l3);
    for(i=0;i<=6;i++){ex++;
            if(i==6) i=0;
            else {}
        if(d[i]==' ') {temp++;//printf(" %d ",temp);
        }
        else {
            count++;//printf(" %d ",count);
            if(count==l3) {d[i]=' '; count=0;}
        }
        if(temp==l3*5) break;
        else {}
    }
   // printf("\n%d\n%s\n%d",ex,d,strlen(d));
   for(i=0;i<6;i++){
    if(d[i]!=' '){
        printf("%c",d[i]);
    }
   }
    return 0;
}

File: flames.java
/*Program for Flames*/

import java.util.*;
class flames{
	static int i,j,count=0,len,temp=0;
	public static void main(String []args){
		Scanner s = new Scanner(System.in);
		String s1=s.next();
		String s2=s.next();
		s1=s1.toLowerCase();
		s2=s2.toLowerCase();
		char[] str1 = new String(s1).toCharArray();
		char[] str2 = new String(s2).toCharArray();
		System.out.println(s1+"  "+s2);
		System.out.println(s1.length()+"  "+s2.length());
		for(i=0;i<s1.length();i++){
			for(j=0;j<s2.length();j++){
				if(str1[i]==str2[j]) {
					count++;
					str1[i]=' ';str2[j]=' ';
					break;
				}
			}
		}
		len=s1.length()+s2.length()-(2*count);
		System.out.println(len);
		
		count=0;
		char[] flames = new String("flames").toCharArray();
		for(i=0;i<=6&&temp!=5;i++){
			if(i==6) i=0;
			if(flames[i]!=' ') 
			{
				count++;
				if(len==count) {
					flames[i]=' '; 
					count=0;
					temp++;
				}
			}
		}
		if(len==0)
			System.out.println("Friends");
		for(i=0;i<6&&len!=0;i++){
			if(flames[i]!=' '){
				switch(flames[i]){
					case 'f':
					System.out.println("Friends");
					break;
					case 'l':
					System.out.println("Love");
					break;
					case 'a':
					System.out.println("Attraction");
					break;
					case 'm':
					System.out.println("Marriage");
					break;
					case 'e':
					System.out.println("Enemy");
					break;
					case 's':
					System.out.println("Sibling");
					break;
				}
			}
		}
	}
}
